types/mongoose/tslint.json

```diff
@@ -1,6 +1,7 @@
 {
     "extends": "dtslint/dt.json",
     "rules": {
+        "newline-per-chained-call": false,
```

Comment:
Should already be disabled in the latest `dtslint/dt.json`

Reply:
fd9caa1bdad5f86c023806f46f2f9c634008fde0 removed